modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java

```diff
@@ -1200,6 +1201,16 @@ private boolean _isExternalRepositorySearch() {
 		return false;
 	}
 
+	private boolean _isRootFolderARootFolder() {
```

Comment:
What about naming this method "_isRepositoryRoot" because it should be true if the folder is root for any repository.

Reply:
better for me!

I will change it, solve the SF problem and rebase it!

Thanks!